src/engine/source/server/CMakeLists.txt

```diff
@@ -54,4 +54,5 @@ target_include_directories(server
     ${RxCpp_SOURCE_DIR}/Rx/v2/src
     ${ENGINE_SERVER_SOURCE_DIR}/base
     ${ENGINE_SOURCE_DIR}/API/include
+    ${ENGINE_SOURCE_DIR}/cmds/src
```

Comment:
Just to understand. Why do we need to include the /cmds/src to the target_include_directories ?

Reply:
we need a file from that directory since it contains the maximum size of events in the queue. If you can't, it throws this error:
fatal error: defaultSettings.hpp: No such file or directory
    26 | #include "defaultSettings.hpp"
       | ^~~~~~~~~~~~~~~~~~~~~